src/pages/Governance/Stake/Index.tsx

```diff
@@ -44,20 +53,20 @@ export function StakePage() {
   };
 
   const onSubmitClick = async () => {
-    // TODO - handle error
-    if (!stakingAmount || !operatorAddr || !voterAddr) return;
-    if (
-      !isValidAccountAddress(operatorAddr) ||
-      !isValidAccountAddress(voterAddr)
-    ) {
-      return;
-    }
     // TODO - this will likely be very dependent per network
     // pull from on chain config 0x1::stake::ValidatorSetConfiguration
     const stakingAmountNumber = parseInt(stakingAmount);
     if (stakingAmountNumber <= 0) return;
```

Comment:
can we also show the error for the staking amount?

Reply:
With the current check `setStakingAmount(event.target.value.replace(/[^0-9]/g, ""));`, I don't think there will ever be negative number inputs. But I'll add error message for `0`. Good catch!

Reply:
also when the staking amount is left empty, `!stakingAmount` should handle both cases I believe